.pending/bugfixes/gaia/3656-distribution-de

```diff
@@ -0,0 +1 @@
+#3656 distribution delegation zero export bugfix
```

Comment:
Hang on, 3656 is https://github.com/cosmos/cosmos-sdk/pull/3656. Is it correct to link the two?